### DISCUSSION TITLE
Control a light group not an individual light

### DISCUSSION THREAD

**Discussion by garagenlager** [EPMatt/awesome-ha-blueprints#196] [Q&A] [answered]
Hi, Ìm successfully operating your blueprint with an IKEA E1743 on a Single light. I World like to control now a group of lights Instead of an Individual light. Is there an Easy Way to Change from a Single light to a group of do I have to add all lights Individually?

**Comment by EPMatt** [comment, marked as answer]
Hi @garagenlager,

thank you for posting your question here. If you're using an automation built with the [E1743 controller blueprint](https://epmatt.github.io/awesome-ha-blueprints/docs/blueprints/controllers/ikea_e1743), linked with an automation built with the [Light Hook blueprint](https://epmatt.github.io/awesome-ha-blueprints/docs/blueprints/hooks/light), you can provide a [Light Group Entity](https://www.home-assistant.io/integrations/light.group) instead of a regular light entity in the `light` input for the Light Hook. 

Alternatively you can build multiple Light Hook automations, each one for every light you want to control with your remote, and link them to the same controller automation. Although I won't suggest this latter approach, it might be useful if you want to provide different settings for each single light source, but having all of them controlled by the same device.

I hope my answer will help you. Please let me know if you have any additional questions or doubts. :)

**Reply by garagenlager** [reply]
Hi, thanks for your quick answer. Light Group entity was exactly what I was looking for. I had grouped the lights throught the UI in groups, but obviously there´s a difference between the UI Groups and the lights group added through explicit  YAML code. By adding light groups through yaml code in the config file, evrything works fine now.

**Reply by EPMatt** [reply]
Hi @garagenlager,

Thank you for your feedback. Great! I'm happy to hear that you were able to configure your setup with light groups. :) 

Happy automating, and enjoy the blueprints! 🚀
